pkgs/servers/mastodon/version.nix

```
@@ -0,0 +1 @@
+"v2.9.3"
```

```suggestion
"2.9.3"
```
For consistency with the rest of Nixpkgs.

---

There are some packages with and some without v in nixpkgs, depending on the upstream tags. In the case of mastodon the upstream tags include a v. If you still think it should be changed, please also change the update script to remove the v when updating the release.

---

Agree this shouldn’t have the v.

---

done